__tests__/app.test.js

```diff
@@ -116,4 +116,51 @@ describe("/api/reviews", () => {
         expect(reviews).toBeSortedBy("created_at", { descending: true });
       });
   });
-}); 
+});
+
+describe("/api/reviews/:review_id/comments", () => {
+  test("Get - status: 200 - return an array of comments of the given review id", () => {
+    return request(app)
+      .get("/api/reviews/2/comments")
+      .expect(200)
+      .then((response) => {
+        const { comments } = response.body;
+
+        comments.forEach((comment) => {
+          expect(comment.hasOwnProperty("comment_id")).toBe(true);
```

Comment:
Can we check the data type of each property. Retrospectively the same comment can be applied to the test on line 88.